File: db4o.net/Db4objects.Db4o.CS/Db4objects.Db4o.CS/CS/Internal/MessageEventArgs.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using System;
using Db4objects.Db4o.CS.Internal.Messages;

namespace Db4objects.Db4o.CS.Internal
{
	public class MessageEventArgs : EventArgs
	{
		private IMessage _message;

		public MessageEventArgs(IMessage message)
		{
			_message = message;
		}

		public virtual IMessage Message
		{
			get
			{
				return _message;
			}
		}
	}
}

File: db4o.cs/src/com/db4o/cs/internal/MessageEventArgs.java
package com.db4o.cs.internal;

import com.db4o.cs.internal.messages.*;
import com.db4o.events.*;

public class MessageEventArgs extends EventArgs {

	private Message _message;

	public MessageEventArgs(Message message) {
	    _message = message;
    }

	/**
	 * @sharpen.property
	 */
	public Message message() {
		return _message;
    }

}
